src/story/story.routes.js

```diff
@@ -4,12 +4,6 @@ var router = express.Router();
 var controller = require('./story.controller');
 var voteController = require('../vote/vote.controller')
 
-//Middle ware that is specific to this router
```

Comment:
You are right. This is unrelated.

I understand we must be cautious introducing modifications to the current feature but in this case, this is dead/unused code.